iceoryx_examples/icedelivery/CMakeLists.txt

```diff
@@ -35,7 +35,7 @@ target_compile_options(iox-cpp-publisher PRIVATE ${ICEORYX_WARNINGS} ${ICEORYX_S
 
 add_executable(iox-cpp-publisher-untyped ./iox_publisher_untyped.cpp)
 target_link_libraries(iox-cpp-publisher-untyped
-    iceoryx_posh::iceoryx_posh
+    iceoryx_posh::iceoryx_posh # @todo #590 Should link against iceoryx_runtime (which depends on iceoryx_posh)
```

Comment:
I thought no todos in code. You could add this todo to the corresponding issue as a todo, then we can keep really track of it.

Reply:
No todos? :grin: There are 127 ` @ todo` in the iceoryx code base currently. I can move this specific one to the issue.

Reply:
I think it's fine to have todos in the code as long as they are linked to an issue like this one

Reply:
Removed as we want to keep the alias `iceoryx_posh` for backwards compatibility and easier migration towards iceoryx `v2.0.0` for users.